.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files

```

Comment:
the .idea directory should not contain a .gitignore file inside it. the .gitignore file should be in the root of the project